fastly/fixtures/managed_logging/recreate.yaml

```
@@ -0,0 +1,51 @@
+---
+version: 1
+interactions:
+- request:
+    body: Kind=1&ServiceID=7i6HN3TK9wS159v2gPAZ8A
+    form:
+      Kind:
+      - "1"
+      ServiceID:
+      - 7i6HN3TK9wS159v2gPAZ8A
+    headers:
+      Content-Type:
+      - application/x-www-form-urlencoded
+      User-Agent:
+      - FastlyGo/2.1.0 (+github.com/fastly/go-fastly; go1.15.5)
+    url: https://api.fastly.com/service/7i6HN3TK9wS159v2gPAZ8A/log_stream/managed/instance_output
+    method: POST
+  response:
+    body: '{"msg":"Event Buffer has already been enabled for this service"}'
```

My understanding is that the production name is now 'Log Tailing':

```suggestion
    body: '{"msg":"Log Tailing has already been enabled for this service"}'
```

---

"Event Buffer" is what is returned from the API currently.

---

OK I guess that might change then by the time of GA maybe? But that's fine 👍🏻